### PR TITLE
Add project module in webhook script2.

### PR DESCRIPTION
**Pull request by Anish-addweb**
Add project module in webhook script2.

### CODE REVIEW

File: administrator-guides/integrations/github/README.md [hunk 1]
```diff
@@ -454,31 +539,32 @@ const githubEvents = {
...
     return {
       content: {
-        icon_url: user.avatar_url,
         alias: user.login,
         text: text,
         attachments: [attachment]
       }
   };
-  },  // End GitHub Push
+  },  // End Github Push
```
Comment (engelgabriel) [inline administrator-guides/integrations/github/README.md]: Please keep GitHub with a capital H, as they prefer like that... see the footer of github.com :)

File: administrator-guides/integrations/github/README.md [hunk 2]
```diff
@@ -255,9 +255,7 @@ This script will generate notifications for the following repository events:
 
 ![Screenshot of messages generated by GitHub integration script 2](github-2.png)
 
-```javascript
-/* exported Script */
-
+```/* exported Script */
```
Comment (engelgabriel) [inline administrator-guides/integrations/github/README.md]: Why was this changed? The docs works better when we define the programming language

**Review by engelgabriel — CHANGES_REQUESTED** [review]

File: administrator-guides/integrations/github/README.md (hunk 1 above)
Comment (Anish-addweb) [reply administrator-guides/integrations/github/README.md]: Ok Done

File: administrator-guides/integrations/github/README.md (hunk 2 above)
Comment (Anish-addweb) [reply administrator-guides/integrations/github/README.md]: Ok i checked your way and it does work better as you mentioned, hence reverting to what you had written.

**Comment by MartinSchoeler** [comment]
@Anish-addweb please check your line endings, make sure you are using LF line endings.

**Comment by MartinSchoeler** [comment]
@Anish-addweb Also could you elaborate a bit what this pull request is about? Thanks!

**Comment by MartinSchoeler** [comment]
@Anish-addweb Could you address my last commentaries? Thanks!

**Comment by MartinSchoeler** [comment]
I am closing this due lack of response, and explanation for what this PR is about. If you decide to go back to this, please reopen this PR. Thanks!
